utility/install_prereq.py

```diff
@@ -1,14 +1,15 @@
-from ceph.utils import setup_repos
 from cli.utilities.packages import Package, SubscriptionManager
-from cli.utilities.utils import get_custom_repo_url, os_major_version
+from cli.utilities.utils import get_builds_by_rhbuild
 from utility.log import Log
-from utility.utils import fetch_build_artifacts
 
-RHEL8_EPEL_RELESE_RPM = (
-    "https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm"
-)
-RHCEPH5_TOOLS_REPO = "rhceph-5-tools-for-rhel-{}-x86_64-rpms"
-RHEL8_ANSIBLE_REPO = "ansible-2.9-for-rhel-8-x86_64-rpms"
+RHCS5_REPOS = {
```

Comment:
One suggestion, can't we add these repos in a separate file rather than within the code? Maybe a 'repositories.yaml/ini' file under conf/inventory or any other location and then append to that file for future releases.

Reply:
@pranavprakash20 agree, need to work on it